documentation/asciidoc/computers/linux_kernel/patching.adoc

```diff
@@ -1,48 +1,66 @@
-== Patching the kernel
+== Patch the kernel
 
-When xref:linux_kernel.adoc#building[building] your custom kernel you may wish to apply patches, or collections of patches (patchsets), to the Linux kernel.
+When building your custom kernel, you may wish to apply patches or collections of patches (patchsets) to the Linux kernel.
 
-Patchsets are often provided with newer hardware as a temporary measure, before the patches are applied to the upstream (mainline) Linux kernel and then propagated down to the Raspberry Pi kernel sources. However, patchsets for other purposes exist, for instance to enable a fully preemptible kernel for real-time usage.
+We often provide patchsets as a temporary measure to support new hardware before the patches make it into the Linux kernel and the Raspberry Pi kernel. However, patchsets for other purposes exist, for instance to enable a fully pre-emptible kernel for real-time usage.
```

Comment:
> We often provide patchsets...

Ummm, no we don't - we always integrate support for new hardware directly into the Raspberry Pi kernel; AFAIK we never distribute patchsets. I believe the previous version of these instructions might have been talking about 3rd-party hardware being supported by temporary patchsets.